packages/extract-react-types/src/index.js

```diff
@@ -1426,7 +1433,7 @@ function getContext(
   filename /*:? string */,
   resolveOptions /*:? Object */
 ) {
-  let plugins = ['jsx'];
+  let plugins = ['jsx', ['decorators', { decoratorsBeforeExport: true }]];
```

Comment:
Hey @bebraw, the flow error you're seeing is coming from this line. The reason is where it is defined in the library we are using [here](https://github.com/babel-utils/babylon-options/blob/daa7f9517623ccc43c2105e043cfdecbdff59444/index.js#L42), it says that plugins must be an array of strings, not recognising this syntax.

If you can add a `/* $FlowFixMe - need to update types in babylon-options */` to the line where plugin is passed in, you should be able to get flow passing, and then we can get this merged in.

Otherwise, these changes look fine.